main/apps.js

```diff
@@ -256,14 +256,30 @@ function readAppInfo(appPath) {
     const packageJsonPath = path.join(appPath, 'package.json');
     return fileUtil.readJsonFile(packageJsonPath)
         .then(packageJson => {
-            const iconPath = path.join(appPath, 'icon.png');
+            const resourcesPath = path.join(appPath, 'resources');
+            let iconPath = path.join(resourcesPath, 'icon.png');
+            if (!fs.existsSync(iconPath)) {
+                iconPath = path.join(appPath, 'icon.png');
+            }
```

Comment:
I see that the icon.png is still used in the window title bar. Is this what is shown in the dock on macOS when launching a shortcut, and in that case does it look good on retina?